src/main/java/network/brightspots/rcv/HartCvrReader.java

```diff
@@ -109,10 +117,31 @@ private void readCastVoteRecord(List<CastVoteRecord> castVoteRecords, Path path)
         ArrayList<Pair<Integer, String>> rankings = new ArrayList<>();
         if (contest.Options != null) {
           for (Option option : contest.Options) {
-            String candidateName = option.Id;
-            if (candidateName.equals(source.getUndeclaredWriteInLabel())) {
-              candidateName = Tabulator.UNDECLARED_WRITE_IN_OUTPUT_LABEL;
+            String candidateCode = option.Id;
+            String candidateName = option.Name;
+            if (candidateCode.equals(source.getUndeclaredWriteInLabel())) {
+              candidateCode = Tabulator.UNDECLARED_WRITE_IN_OUTPUT_LABEL;
+            }
+            if (this.candidateCodesToCandidates.containsKey(candidateCode)) {
+              if (!this.candidateCodesToCandidates.get(candidateCode).Name.equals(candidateName)) {
+                String message = """
+                        Candidate Code %s associated with more than one candidate name.
+                        Originally associated with name '%s'.
+                        In CVR at '%s' it is associated with '%s'.
+                        """.formatted(candidateCode,
```

Comment:
This syntax will cause the error message to have a ton of spaces throughout the error message. Using `+`s will avoid that:
```
                String message =
                        "Candidate Code %s associated with more than one candidate name.  " +
                        "Originally associated with name '%s'. " + 
                        "In CVR at '%s' it is associated with '%s'."
                        .formatted(candidateCode,
```